<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClientDialog</class>
 <widget class="QDialog" name="AddClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter client</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QFormLayout" name="nameLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Nom *</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Doit commencer par une majuscule</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhLatinOnly</set>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="firstNameLabel">
         <property name="text">
          <string>Prénom *</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="firstNameLineEdit">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Doit commencer par une majuscule</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhLatinOnly</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="addressLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>Adresse *</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addressLineEdit">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cityLabel">
         <property name="text">
          <string>Ville *</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cityLineEdit">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Doit commencer par une majuscule</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhLatinOnly</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="postalCodeLabel">
         <property name="text">
          <string>Code Postal *</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="postalCodeLineEdit">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Code postal au format français</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="maxLength">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="phoneNumLabel">
         <property name="text">
          <string>Num. Tel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="phoneNumLineEdit">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Numéro de téléphone au format français</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rdvLayout">
       <item>
        <widget class="QLabel" name="rdvDayLabel">
         <property name="text">
          <string>Jour du rendez-vous *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCalendarWidget" name="appointmentDayCalendar">
         <property name="toolTip">
          <string>Sélectionnez le jour du RDV</string>
         </property>
         <property name="minimumDate">
          <date>
           <year>1752</year>
           <month>9</month>
           <day>15</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="priorityLabel">
           <property name="text">
            <string>Priorité *</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="prioritySpinBox">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Priorité du RDV</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="rdvDuration">
           <property name="text">
            <string>Durée RDV *</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="rdvDurationSpinBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Durée du RDV</string>
           </property>
           <property name="suffix">
            <string> min</string>
           </property>
           <property name="maximum">
            <number>1440</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="listLabel">
       <property name="text">
        <string>Identifiants des ressources *</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="resourcesListView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addResourcesButton">
         <property name="toolTip">
          <string>Ajouter des ressources dans la liste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/add</normaloff>:/add</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveUpRessourcesButton">
         <property name="toolTip">
          <string>Remonter la ressource dans la liste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/moveDown</normaloff>:/moveDown</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveDownRessourcesButton">
         <property name="toolTip">
          <string>Redescendre la ressource dans la liste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/moveUp</normaloff>:/moveUp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRessourcesButton">
         <property name="toolTip">
          <string>Supprimer la ressource de la liste</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ressources.qrc">
           <normaloff>:/delete</normaloff>:/delete</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="commentsLabel">
       <property name="text">
        <string>Commentaires</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="commentTextEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="toolTip">
        <string>Valider l'ajout du client</string>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Annuler l'ajout du client</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="compulsoryInputsLabel">
     <property name="text">
      <string>* Champs obligatoires</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>firstNameLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>cityLineEdit</tabstop>
  <tabstop>postalCodeLineEdit</tabstop>
  <tabstop>phoneNumLineEdit</tabstop>
  <tabstop>appointmentDayCalendar</tabstop>
  <tabstop>rdvDurationSpinBox</tabstop>
  <tabstop>prioritySpinBox</tabstop>
  <tabstop>resourcesListView</tabstop>
  <tabstop>addResourcesButton</tabstop>
  <tabstop>moveUpRessourcesButton</tabstop>
  <tabstop>moveDownRessourcesButton</tabstop>
  <tabstop>deleteRessourcesButton</tabstop>
  <tabstop>commentTextEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>checkBeforeSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addResourcesButton</sender>
   <signal>clicked()</signal>
   <receiver>AddClientDialog</receiver>
   <slot>addResources()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkBeforeSubmit()</slot>
  <slot>addResources()</slot>
 </slots>
</ui>
